memote/utils.py

```diff
@@ -138,6 +138,14 @@ def get_ids_and_bounds(iterable):
             elem in iterable]
 
 
+def filter_none(attribute, default):
```

Comment:
This function is superfluous. Use `getattr(object, "attribute", default)` instead.

Reply:
If I remember correctly, we talked about this briefly. I couldn't just use `getattr` since it checks for the presence of the attribute on the object. Even though the attribute may be missing (because optional) in SBML, cobrapy will instantiate all objects including the attributes, but defaults to giving them `None` values. For instance:

xml```
<species boundaryCondition="false" compartment="c" constant="false" fbc:chemicalFormula="C5H9O8P" hasOnlySubstanceUnits="false" id="M_r5p_c" metaid="M_r5p_c" sboTerm="SBO:0000247">
</species>
```

seems to result in something along these lines in the loaded cobrapy model object
```
model.metabolites.r5p_c.name
>
```

Reply:
Right, my bad.